packages/binary-install/index.js

```diff
@@ -27,29 +27,50 @@ class Binary {
       errors.push("name must be a string");
     }
 
+    if (version && typeof version !== "string") {
+      errors.push("version must be a string");
+    }
```

Comment:
I think this might group well logically as an `else if` with the `!version` check below (and you can get rid of the `version &&` check). Just a nit, this is fine as is.

Reply:
oof thanks